.github/workflows/upgrade-liff-sdk.yml

```diff
@@ -0,0 +1,31 @@
+name: Upgrade LIFF SDK
+
+on:
+  schedule:
+    - cron: '0 10 * * 1'
```

Comment:
Can we reduce more? Because we almost only release once a month.

Reply:
Ok